src/components/MCQuestion.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import TextField from "@material-ui/core/TextField";
+import FormControl from "@material-ui/core/FormControl";
+import FormLabel from "@material-ui/core/FormLabel";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+
+// Other option, they type it in
+class MCQuestion extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
I have both state and props in this component (bad). Would be great if someone could help me figure out how to just keep one :)